newyork.css

```diff
@@ -5,14 +5,21 @@ body{
 	
 }
 
```

Comment:
## Name change of file suggestion.
- [ ] What do you think about my proposal to rename this file to `index.css`?